.env.example

```diff
@@ -13,20 +14,27 @@ DB_USERNAME=homestead
 DB_PASSWORD=secret
 
 CACHE_DRIVER=redis
+FILESYSTEM_DRIVER=public
+QUEUE_CONNECTION=sync
 SESSION_DRIVER=redis
-STORAGE_DRIVER=public
-QUEUE_DRIVER=sync
 
-REDIS_HOST=127.0.0.1
-REDIS_PASSWORD=null
-REDIS_PORT=6379
+REDIS_URL=redis://h:null@127.0.0.1:6379
```

Comment:
See [this comment](https://github.com/DoSomething/rogue/pull/1071/files#r461733624).